packages/scss/src/components/_grid.scss

```diff
@@ -131,7 +131,7 @@
 
 /* SM - Columns */
 
-@media only screen and (min-width: $sm) {
+@media only screen and (min-width: _get($luTheme, "breakpoints.sm")) {
 	.grid-sm {
 		flex-basis: 0;
```

Comment:
I've read using the `flex: none | [ <'flex-grow'> <'flex-shrink'>? || <'flex-basis'> ]` shorthand rather than `flex-grow`, `flex-shrink`... properties is recommended.